src/magic_uv/ui/IMAGE_MT_uvs.py

```diff
@@ -118,7 +118,7 @@ def draw(self, context):
         ops.group = sc.muv_align_uv_snap_point_group
         ops.target = sc.muv_align_uv_snap_point_target
 
-        ops = layout.operator(MUV_OT_AlignUV_SnapToEdge, text="Snap to Edge")
+        ops = layout.operator(MUV_OT_AlignUV_SnapToEdge.bl_idname, text="Snap to Edge")
```

Comment:
pep8 test is failed.
Could you apply this suggestion?

```suggestion
        ops = layout.operator(MUV_OT_AlignUV_SnapToEdge.bl_idname,
                              text="Snap to Edge")
```